### ISSUE TITLE
(Feature request) Playlist

### ISSUE THREAD

**Issue by hardBSDk** [essej/sonobus#259]
Allow multiple files to be loaded and queued for playback.

Related to https://github.com/sonosaurus/sonobus/issues/48

**Comment by islisis** [comment]
Yes, a basic feature for any kind of broadcast. On Android too, with support for mp3, please!
